examples/r-job/.saturn/saturn.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "example-r-job",
+  "image_uri": "saturncloud/saturn-rstudio:2021.11.10",
+  "description": "Run a job on Saturn Cloud using R .",
+  "working_directory": "/home/jovyan/git-repos/examples/examples/r-job",
+  "start_script": "sh /home/jovyan/git-repos/Dashboard/startup-scriptJ.sh",
```

Rather than having a start script that's a file with just one line, let's have the start script just directly run the call. Also I renamed it since there might be things we want to do in there besides just install packages

```suggestion
  "start_script": "Rscript setup.R",
```